docs/guide/seo.rst

```diff
@@ -0,0 +1,63 @@
+==================================================
+Search engine optimization (SEO) for documentation
+==================================================
+
+What is search engine optimization?
+-----------------------------------
+
+Search engines drive a significant portion of site traffic for docs that publish as public-facing websites.
+Search engine optimization (SEO) describes the methods used to improve the rank of your web pages in search results for a given set of search terms.
+
+To perform SEO, you must be familiar with the ever-changing algorithms that search engines use to select and arrange the webpages returned for a given set of search terms.
+These algorithms change frequently, so the requirements for achieving the best optimization also change frequently.
+
+SEO for documentation
+---------------------
+
+Good tech writing follows most of the standard practices required for excellent SEO:
+
+- Write imperative, user-centric, goal-oriented headings
+- Make your docs skimmable by breaking content into sections under appropriate headings
+- Focus on good information architecture that provides quality navigation and site hierarchy
+- Include page metadata, such as last modified date, in the ``<head>`` section of your web page (how to do this varies by your publication toolchain)
+- Make sure your documentation meets :doc:`standards for accessibility </guide/writing/accessibility>`
+
+SEO resources
+-------------
+
+Search engine documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Review the search engine documentation guides:
+
+- Google's `Search Central <https://developers.google.com/search/docs>`__
+- Google's `How Search Works <https://www.google.com/search/howsearchworks/how-search-works/ranking-results/>`_ guide
+- Bing's `Webmaster Guidelines <https://www.bing.com/webmasters/help/webmasters-guidelines-30fba23a>`__
+
+WTD resources on SEO
+~~~~~~~~~~~~~~~~~~~~
+
+- :ref:`Topics archive on search <topics-archive-search>`
+
+Books
+~~~~~
+
+- `SEO for Everyone <https://abookapart.com/products/seo-for-everyone>`__, by Rebekah Baggs & Chris Corak
+
+Blogs and websites
+~~~~~~~~~~~~~~~~~~
+
+- `Moz SEO Beginners Guide <https://moz.com/beginners-guide-to-seo>`__
+- `The Ultimate Guide to SEO Meta Tags <https://moz.com/blog/the-ultimate-guide-to-seo-meta-tags>`__ by Moz
+- `Search Engine Land <https://searchengineland.com/>`__
+- `Hubspot's list of SEO blogs <https://blog.hubspot.com/marketing/best-seo-blogs>`__
+- `Search Engine Optimization: A beginner’s guide from Ahrefs <https://ahrefs.com/seo>`__
+- `SEO Marketing Hub 2.0 from Backlinko <http://backlinko.com/hub/seo`__
```

Comment:
Guess I was wrong. Maybe this one?
```suggestion
- `SEO Marketing Hub 2.0 from Backlinko <http://backlinko.com/hub/seo>`__
```

Reply:
Well, `<head>` was still wrong.
This one is too, fixing it.

Thanks again for your good eye.